spinup/run.py

```diff
@@ -153,9 +153,13 @@ def process(arg):
 
     # Special handling for environment: make sure that env_name is a real,
     # registered gym environment.
-    valid_envs = [e.id for e in list(gym.envs.registry.all())]
     assert 'env_name' in arg_dict, \
         friendly_err("You did not give a value for --env_name! Add one and try again.")
+    for env_name in arg_dict['env_name']:
+        if 'BulletEnv' in env_name:
+            import pybullet_envs
+            break
```

Comment:
no need 'break'

Reply:
I don't understand why there can specify more than one environment. But the "break" just to let us no need to check more, in case for there are more than one name of pybullet environment in arg_dict.

Reply:
yes, you are right.